+bids/File.m

```diff
@@ -65,6 +65,65 @@
   %
   %   file = bids.File(name_spec, 'use_schema', true);
   %
+  % Load metadata (supporting inheritance).
+  %
+  % .. code-block:: matlab
+  %
+  %   f = bids.File('tests/data/synthetic/sub-01/anat/sub-01_T1w.nii.gz');
+  %
```

Comment:
need to check but maybe better to put the help section after each method than all of them at the top

if the file has all the method help at the top at the moment, then let's take care of that in a separate PR.

Reply:
It's more an example, making aware that such functions exists. The actual doc are already in the functions below.

I think it's worth to keep the doc as it is -- long descriptions are common in matlab.

Reply:
OK that makes sense. good with me